optimum/exporters/onnx/model_configs.py

```diff
@@ -632,7 +632,32 @@ def generate_dummy_inputs(self, framework: str = "pt", **kwargs):
         return dummy_inputs
 
 
-class VaeOnnxConfig(ViTOnnxConfig):
+class VaeEncoderOnnxConfig(ViTOnnxConfig):
+    ATOL_FOR_VALIDATION = 1e-2
+    DEFAULT_ONNX_OPSET = 14
+
+    NORMALIZED_CONFIG_CLASS = NormalizedConfig.with_args(
+        num_channels="in_channels",
+        image_size="sample_size",
+        allow_new=True,
+    )
+
+    DUMMY_INPUT_GENERATOR_CLASSES = (DummyVisionInputGenerator,)
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return {
+            "sample": {0: "batch_size", 1: "num_channels", 2: "height", 3: "width"},
+        }
+
+    @property
+    def outputs(self) -> Mapping[str, Mapping[int, str]]:
+        return {
+            "latent_sample": {0: "batch_size", 1: "num_channels_latent", 2: "height_latent", 3: "width_latent"},
+        }
+
+
+class VaeDecoderOnnxConfig(ViTOnnxConfig):
     ATOL_FOR_VALIDATION = 1e-3
     DEFAULT_ONNX_OPSET = 14
```

Comment:
Could you add a comment as to why the opset 13, 12 fail?

Reply:
The text encoder (a `CLIPTextModel`) needs a min opset  of 14 to support the [Trilu](https://github.com/onnx/onnx/blob/main/docs/Operators.md#Trilu) operator. Because we export all components together, I've set the same default opset to every components

Reply:
Could you add this as a comment?